proto/beacon/p2p/v1/types.proto

```diff
@@ -73,10 +73,10 @@ message PendingAttestationRecord {
 }
 
 message AttestationRecord {
-  AttestationData data = 1;
-  bytes participation_bitfield = 2;
-  bytes custody_bitfield = 3;
-  bytes aggregate_signature = 4; // Type of [uint384] ?
+  AttestationData data = 1 [deprecated=true];
+  bytes participation_bitfield = 2 [deprecated=true];
+  bytes custody_bitfield = 3 [deprecated=true];
+  bytes aggregate_signature = 4 [deprecated=true]; // Type of [uint384] ?
```

Comment:
Can we ranges for attestations? im afraid this proto could change a lot so we should use similar structure to beacon state

Reply:
Message type `AttestationRecord` is soon going to be replaced by `Attestation` thought. Do you want me to add ranges for message `Attestation`?

Reply:
Can you just rename AttestationRecord to Attestation? The look like the same object.
If it requires code changes, please follow up immediately with another PR since it should be a trivial change.

Reply:
The test will fail. This requires code change I will address this in the subsequent PR.